portfolio.html

```diff
@@ -1,16 +1,17 @@
 <html lang="en">
 <head>
 <title>Tom Hughes | Portfolio</title>
-<meta name="viewport" content="width=device-wdith, initial-scale=1.0">
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
 <link rel="stylesheet" type="text/css" href="normalize.css">	
 <link rel="stylesheet" type="text/css" href="portfoliostyle.css"> 
+<script src="https://kit.fontawesome.com/05d75d114c.js" crossorigin="anonymous"></script>
 </head>	
 <body>
 	<nav>
 		<div class="anchors">
-			<a href="#">About</a>
-			<a href="#">Portfolio</a>
-			 <a href="#">Socials</a>
+			<a class ="link" href="#">About</a>
```

Comment:
links to scroll to different parts of page don't work yet. Super easy fix (#"IDnamegoeshere") + in the CSS 
html {
scroll-behaviour: smooth; 
}

I also think you should use more semantic names i.e. "nav_link"